docs/debugging.rst

```diff
@@ -94,6 +77,26 @@ For now, Taichi-scope ``print`` supports string, scalar, vector, and matrix expr
 
     Please note that host access or program end will also implicitly invoke ``ti.sync()``.
 
+.. note::
+
+    Note that ``print`` in Taichi-scope can only receive **comma-separated parameter**. Neither f-string nor formatted string should be used. For example:
+
+    .. code-block:: python
+
```

Comment:
Note: `rst` consider two line without a newline between then as the **same line**, i.e.:

```
hello
world
```

is same as

```
hello world
```

So you want to use:
```
hello

world
```
to get the desired result.